xbmc/addons/addoninfo/AddonInfo.cpp

```diff
@@ -243,4 +258,51 @@ const std::string& CAddonInfo::GetTranslatedText(const std::unordered_map<std::s
   return StringUtils::Empty;
 }
 
+bool CAddonInfo::SupportsMultipleInstances() const
+{
+  switch (m_addonInstanceUseType)
+  {
+    case AddonInstanceUse::SUPPORTED_MANDATORY:
+    case AddonInstanceUse::SUPPORTED_OPTIONAL:
+      return true;
+    case AddonInstanceUse::SUPPORTED_BY_SETTINGS:
+      return m_supportsInstanceSettings;
+    case AddonInstanceUse::NONE:
+    default:
+      return false;
+  }
+}
+
+std::vector<uint32_t> CAddonInfo::GetKnownInstanceIds() const
+{
+  using namespace XFILE;
```

Comment:
As you are using a type from this namespace only one time in this method, this line has more characters than prefix of the one affected line `XFILE::`. :-)